.gitlab-ci.yml

```diff
@@ -188,6 +192,17 @@ check_new_flaky_tests:
       when: never
     - when: on_success
 
+pr_name_lint:
+  stage: tests
+  needs: ["run-tests-trigger"]
+  extends: .testrunner
+  script:
+    - pip install gitlint
+    - echo $GH_PR_TITLE | gitlint
+  rules:
+    - if: "$GH_PR_TITLE" != ""
```

Comment:
I wonder if we have this `$HAS_OPEN_PR == "true"` because what if something fails and we accidentally end up with a `GH_PR_TITLE=""` due to the pipeline variable job?